confidential-data-hub/golang/pkg/api/cdhttrpc/cdhttrpc.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
```

Comment:
same as mentioned. We use the same proto defined in https://github.com/confidential-containers/guest-components/blob/main/confidential-data-hub/hub/protos/api.proto

Reply:
Done.